src/v/json/json.h

```diff
@@ -95,6 +95,17 @@ void rjson_serialize(
     w.EndArray();
 }
 
+template<typename T, size_t max_fragment_size = 8192>
```

Comment:
nit: does it make sense to have the `max_frag_size` template defaulted to something?

Reply:
No i'll change that